security/certificate_types_descriptions/machine-config-operator-certificates.adoc

```diff
@@ -27,7 +27,7 @@ include::snippets/mcs-endpoint-limitation.adoc[]
 
 * xref:../../machine_configuration/index.adoc#machine-config-operator_machine-config-overview[Machine Config Operator].
 
-* xref:../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[About the OpenShift SDN network plugin].
+* xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[About the OVN-Kubernetes network plugin]
```

Comment:
```suggestion
* xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[About the OVN-Kubernetes network plugin]
```
Perhaps note this for later because it is in a module name..."plugin".